src/react/components/shell/mainContentRouter.test.tsx

```diff
@@ -1,8 +1,9 @@
 import { shallow, mount, ReactWrapper } from "enzyme";
 
 import React from "react";
-import { Route, StaticRouter as Router } from "react-router-dom";
-
+import { Route, StaticRouter as Router, Redirect } from "react-router-dom";
+import SignedInRoute from "./signedInRoute";
+import SignedOutRoute from "./signedOutRoute";
```

Comment:
renommer SignedOutRoute en AnonymousRoute

Reply:
I have always seen a signed-out user as anonymous user, except when it has just signed-out, and that there is a special treatment following this new state.

In our case, it is more about any one who is not signed-in, hence anonymous

does that make sense ?

Reply:
Okay, now it makes sense :)